### PR TITLE
fix manifest

### LINKED ISSUE #1158: Firefox Nightly: Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”)
## Issue and Steps to Reproduce

  - Try search on google
  - Get Ember inspector addon error in browser console
  - And can't see google search results

Links:
https://www.google.com/search?newwindow=1&client=firefox-b-d&sxsrf=ALeKk03nSJjNh6tFcy8xukH0BWNDjzapHQ:1584126596986&q=meu.inss.gov.br+login&sa=X&ved=2ahUKEwjKq964k5joAhViK7kGHTyHBF4Q1QIoAXoECAEQAg

## Your environment

Add-on: 3.13.2
Firefox:
Update Channel 	nightly
User Agent 	Mozilla/5.0 (Windows NT 10.0; rv:76.0) Gecko/20100101 Firefox/76.0

## Screenshots
error-boot-js-ember-inspector-csp

### PR DESCRIPTION
**Pull request by patricklx**
fixes #1158

### CODE REVIEW

**Comment by RobbieTheWagner** [comment]
@patricklx was `matches` all we were missing then? What exactly does this do?

**Comment by patricklx** [comment]
i'm not sure 😅
thats from https://developer.chrome.com/docs/extensions/migrating/manifest/#update-wa-resources
and the error reported when loading the extension locally into chrome

**Comment by patricklx** [comment]
yes, it loads the extension now, but it fails to load on any page...
some csp issue

**Comment by patricklx** [comment]
I understand now. adding inline scripts directly is not allowed any more. But its possible to load scripts that are specified in the allowed resourced inside the manifest.
I will add fixes for that as well here

File: skeletons/web-extension/content-script.js [hunk 1]
```diff
@@ -90,7 +90,7 @@
       if (!injected) {
         // cannot use eval here, as the context is limited to the content script-
         const elem = document.createElement('script') ;
-        elem.textContent = message.value;
+        elem.src =  chrome.runtime.getURL('panes-3-16-0/ember_debug.js');
```
Comment (RobbieTheWagner) [inline skeletons/web-extension/content-script.js:93]: Will specifically loading 3-16-0 here break our check where we switch to different versions for older Ember?

Comment (patricklx) [reply skeletons/web-extension/content-script.js:93]: Yes, just testing now. I will change it later

**Comment by patricklx** [comment]
I think thizis ready.
Let me squash all commits

**Comment by patricklx** [comment]
Squashed and ready

**Comment by patricklx** [comment]
this might also fix #1158

**Comment by RobbieTheWagner** [comment]
@patricklx have you checked to make sure the older versions still boot up for Ember < 3.16?

**Review by RobbieTheWagner — APPROVED** [review]
This looks good to me! Thank you so much for your help 🎉. I just want to make sure we test these changes against some older Ember versions, to make sure loading the old versions works still.

**Comment by patricklx** [comment]
finally found a website...
http://altrim.github.io/ember-cli-tooltipster/ is using Ember  2.10.1
it loads correctly

**Pull request merged**